src/tabs/common/clipboard.ts

```diff
@@ -81,14 +101,12 @@ const copyEquation = (event: MouseEvent) => {
 }
 
 export const initializeCopyToClipboard = () => {
-  const copyableElements = Array.from(document.querySelectorAll('.clickable'))
-  copyableElements.forEach(element => element.addEventListener('click', (event: MouseEvent) => { copyTextContent(event, 'copying_box_kbd') } ))
-
-  const copyableCodeElements = Array.from(document.querySelectorAll('.code-clickable'))
-  copyableCodeElements.forEach(element => element.addEventListener('click', (event: MouseEvent) => { copyTextContent(event, 'copying_box_code') } ))
-
-  const copyableCodeOutputElements = Array.from(document.querySelectorAll('.code-output-clickable'))
-  copyableCodeOutputElements.forEach(element => element.addEventListener('click', (event: MouseEvent) => { copyTextContent(event, 'copying_box_code_output') } ))
+  const copyableElements: Element[] = [].concat(
+    Array.from(document.querySelectorAll('.clickable')),
+    Array.from(document.querySelectorAll('.code-clickable')),
+    Array.from(document.querySelectorAll('.code-output-clickable')),
+  )
+  copyableElements.forEach(element => element.addEventListener('click', copyTextContent ))
```

Comment:
Tarkotin siis että ton yllä oilevan voi sit vielä korvata tämmösellä:
```
document.body.addEventListener('click', copyTextContent)
```
tai `document.body` tilalla joku tarkempi selektori, jonka alla noi kaikki on, tyyliin `document.querySelector('.myContainer', copyTextContent)`

Näin ollen ei tartte jokaista elementtiä erikseen hakea ja sitten määritellä jokaiselle omaa click-handleria.